examples/io/plot_read_events.py

```diff
@@ -0,0 +1,77 @@
+"""
+.. _ex_read_events:
```

Comment:
also this is consistent with what we do in the tutorials (e.g. add `.. tut_shorthand_name:` at the beginning)